pkg/client/dtclient/settings_client.go

```diff
@@ -41,6 +42,13 @@ import (
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/config/coordinate"
 )
 
+// SettingsResourceContext.Operations possibilities
+const (
+	DeleteOperation = "delete"
+	WriteOperation  = "write"
+	//ReadOperation   = "read"
+)
+
 // DownloadSettingsObject is the response type for the ListSettings operation
 type DownloadSettingsObject struct {
```

Comment:
Why not move to a separate file

Reply:
I would suggest doing it in a separate ticket when we rename `dtclinet` folder to `settings` as the call in code would also be `settings.DownloadSettingsObject`. If you have a strong opinion on this I can do it now, but it will be a much bigger PR then.